tools/data_source/upload.py

```diff
@@ -5,6 +5,7 @@
 # to be reflected in galaxy.web.controllers.tool_runner and galaxy.tools
 from __future__ import print_function
 
+import logging
```

Comment:
This is triggering the `flake8` I100 message which is causing build failures. From https://pypi.python.org/pypi/flake8-import-order:

"In general stdlib comes first, then 3rd party, then local packages, and that each group is individually alphabetized" - so that's

````
import codecs
import gzip
import logging
import os
````